dropwizard-testing/pom.xml

```diff
@@ -63,6 +63,12 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
+            <groupId>javax.transaction</groupId>
+            <artifactId>jta</artifactId>
+            <version>1.1</version>
```

Comment:
Oh, right! I forgot about the differences between JTA 1.1 and JTA 1.2. I guess we need to remove JTA 1.1 from `jackson-datatype-hibernate5` then.